docs/assets/style.css

```
@@ -0,0 +1,1279 @@
+h6 {
+  font-size: 0.875rem;
+  margin: 2.33rem 0;
+}
+
+.uppercase {
+  text-transform: uppercase;
+}
+
+pre {
+  white-space: pre;
+  white-space: pre-wrap;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

white-space is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)